docs/usage.md

```diff
@@ -62,10 +62,48 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Performing git actions
```

Comment:
Depending on how this gets merged with the other in flight PR, this title might not be accurate. Something like "Handling git branches" might be more accurate as this section does not allow for performing all git actions.